### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pawelborkar**

### CODE REVIEW

**Comment by PawelBorkar** [comment]
Please review the PR

File: README.md [hunk 1]
```diff
@@ -1,47 +1,83 @@
...
+
+<hr>
+  <p>
+  <br>
+  <strong>Manage all of your organization's APIs in Postman, with the industry's most complete API development environment.</strong>
+</p>
+
+
+<p align="center">
+  <a href="https://www.getpostman.com"><strong>www.getpostman.com</strong></a>
```
Comment (DannyDainton) [inline README.md:31]: Can you change this to `www.postman.com` while you're editing the readme :D

File: LICENSE.md [hunk 2]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Postdot Technologies, Inc
+   Copyright 2021 Postdot Technologies, Inc.
```
Comment (codenirvana) [inline LICENSE.md:190]: Ref: https://github.com/postmanlabs/newman/pull/2627#pullrequestreview-609939416

File: README.md [hunk 3]
```diff
@@ -1,47 +1,83 @@
...
-8. [Using Newman with the Postman API](#using-newman-with-the-postman-api)
-9. [Using Newman in Docker](#using-newman-in-docker)
-10. [Using Socks Proxy](#using-socks-proxy)
-11. [Migration Guide](#migration-guide)
-12. [Compatibility](#compatibility)
-13. [Contributing](#contributing)
-14. [Community Support](#community-support)
-15. [License](#license)
+
+  - [Table of contents](#table-of-contents)
```
Comment (codenirvana) [inline README.md:39]: Wondering why it's changed to an unordered list?

**Review by codenirvana — CHANGES_REQUESTED** [review]
Great effort, like the center-aligned header but I don't feel the rest of the changes are required at the moment.

File: README.md (hunk 3 above)
Comment (DannyDainton) [reply README.md:39]: @codenirvana Does it _need_ to be an ordered list? 

Personally, I feel it looks a lot cleaner without the numbers. 😄

**Comment by shamasis** [comment]
The thing with this is, with the improved aesthetics, we are loosing out on easy update to README.

This isn't a hard input, but my opinion is that it is better to stick to islands of HTML to format specific things than to make HTML a core need to craft the README.

I mean, at this moment, we are better off with more documentation around things than the aesthetics of the documentation that may hinder crafting more docs.

**Comment by PawelBorkar** [comment]
Is there any chance of getting this PR merged?

**Review by shamasis — CHANGES_REQUESTED** [review]
Not in favour of too much HTML in markdown.

**Pull request closed without merging**